projects/ng-core-tester/src/assets/i18n/de.json

```diff
@@ -26,5 +26,25 @@
   "Add mode": "hinzufügenmodus",
   "Please select a type": "Bitte wählen Sie einen Typ",
   "Full-text": "Volltext",
-  "Main title": "Haupttitel"
+  "Main title": "Haupttitel",
```

Comment:
translations should be done at the release time...

Reply:
It's the translations that have to be tested to make sure they work on the language switchover.